osf/migrations/0061_auto_20171002_1438.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-10-02 19:38
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+from django.core.paginator import Paginator
+
+from osf.models import OSFUser
+from osf.models import NotificationSubscription
+from website.notifications.utils import to_subscription_key
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def add_reviews_notification_setting(*args, **kwargs):
+    active_users = OSFUser.objects.filter(date_confirmed__isnull=False).exclude(date_disabled__isnull=False).exclude(is_active=False).order_by('id')
+    total_active_users = active_users.count()
+
+    logger.info('About to add a global_reviews setting for {} users.'.format(total_active_users))
+
+    paginated_users = Paginator(active_users, 1000)
```

Comment:
Is using `Paginator` better than `active_users.iterator()`?

Reply:
No idea, I was just copying another migration that used Paginator.

Reply:
If we could bulk-create `NotificationSubscription` and the many-to-many, one batch per page, then `Paginator` would make sense. I think we should either do that or, if it's too messy, use `.iterator()`